packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { IStyle, DefaultPalette } from '../../../Styling';
+
+export interface IBeakStylesProps {
+  left?: string | null;
+  top?: string | null;
+  width?: string;
+  height?: string;
+}
+
+export interface IBeakStyles {
+  /**
+   * Style for the root element in the default enabled/unchecked state.
+   */
+  root?: IStyle;
+  beak?: IStyle;
+}
+
+export function getStyles(props: IBeakStylesProps): IBeakStyles {
+  return {
+    root: [
+      {
+        position: 'absolute',
+        boxShadow: 'inherit',
+        border: 'inherit',
+        boxSizing: 'border-box',
+        transform: 'translate(-50%, -50%)',
```

Comment:
Did you mean to rotate? Right now the border will look like a square.

Reply:
Actually the beak is now a real triangle dynamically made with an SVG

Reply:
Without the border you'll need to have the SVG handle the outline somehow, otherwise it will just be the fill color.